File: workspace/solutions/kotlin/lib/src/test/kotlin/s2/s1/s0/s4/s_2104_test.kt
package s2.s1.s0.s4

import utils.T
import kotlin.test.Test
import s2.s1.s0.s4.s_2104_00.Solution as s_2104_00
import s2.s1.s0.s4.s_2104_01.Solution as s_2104_01

class s_2104_test {
    @Test
    fun test() {
        val t = T(
            this::class,
            s_2104_00::class,
            s_2104_01::class,
        )
        t.run()
    }
}

File: workspace/solutions/rust/src/s2/s1/s0/s4/s_2104_test.rs
#[cfg(test)]
mod tests {
    use super::super::s_2104_00;
    use super::super::s_2104_01;
    use crate::utils::{json::ValueExtension, test::t};

    #[test]
    fn test() -> () {
        t(
            module_path!(), //
            &vec![
                s_2104_00::SOLUTION, //
                s_2104_01::SOLUTION,
            ],
            |solve, input| {
                serde_json::Value::from(solve(
                    input[0].to_i32_array().unwrap(), //
                ))
            },
        );
    }
}
